engines/rancher_on_eks/app/views/rancher_on_eks/steps/index.html.haml

```diff
@@ -26,5 +26,16 @@
     %div.form-group
       = form.submit t('actions.exec'), class: 'btn btn-primary'
 -# end
+
+- if @resources.length > 0
+  %p= t('engines.rancher_on_eks.wrapup.resources.caption')
+  %ul
+    - @resources.each do |resource|
+      %li
+        = resource.type
+        = resource.id
+    -# end
+-# end
```

Comment:
This ends up on the wrapup page. But we want to display the information on  the "deployment" page in case something goes wrong and the user cannot get to the wrapup page.

Reply:
I don't follow, the information is in the "deployment" page ? 
that index.html.haml is the deployment page, showing the resources created